airflow/models/dag.py

```diff
@@ -317,6 +321,7 @@ def __init__(
         self.is_paused_upon_creation = is_paused_upon_creation
 
         self.jinja_environment_kwargs = jinja_environment_kwargs
+        self.tags = tags
```

Comment:
Should we add logic to remove duplicate tags? Example: `['example', 'example']`

Reply:
See `get_dagtags` function, it convert the tags list (if not None) to a set.

Reply:
Awesome